overrides/app/pages/marketing-pages/index.jsx

```diff
@@ -15,6 +17,11 @@ export const MarketingPage = () => {
 
     const urlPath = location.pathname
 
+    const {req} = useServerContext()
+    if (req) {
+        setIvm(req.ivm)
+    }
```

Comment:
import `setIvm` from `'@builder.io/sdk-react/node/setIvm'`, and call it while passing it `req.ivm` coming from `useServerContext()`